bangazonapi/views/profile.py

```diff
@@ -156,7 +156,7 @@ def cart(self, request):
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={'request': request}).data
-                cart["order"]["line_items"] = line_items.data
+                cart["order"]["lineitems"] = line_items.data
```

Comment:
`cart.order.lineitems` is created in the OrderSerializer, so it's best to get rid of this code since it is redundant. 1) let the serializer be responsible for creating the order object 2) if someone changed the serializer in the future without realizing this code is here it could cause problems

Reply:
made the changes and removed the line